lua/lspconfig/server_configurations/eslint.lua

```diff
@@ -64,7 +64,7 @@ return {
     -- Refer to https://github.com/Microsoft/vscode-eslint#settings-options for documentation.
     settings = {
       validate = 'on',
-      packageManager = 'npm',
+      packageManager = nil,
```

Comment:
if nil can be remove

Reply:
I suppose keeping this as an explicit field can help with the typing, so we can have `nil|"npm"|"yarn"|"pnpm"`?

Reply:
fair